guides/app-guides/borrow-ageur.md

```diff
@@ -0,0 +1,110 @@
+---
+description: How to borrow agEUR from the Angle App
+---
+
+# Borrowing agEUR 
+
+The [Angle Borrowing module](/borrowing-module/README.md) lets you borrow agEUR or get leverage on your crypto directly from the [Angle App](https://app.angle.money/#/borrow). 
+
+Essentially, you can deposit your crypto into what is called a vault, to get a stablecoins loan in exchange. If the value of the crypto you deposited goes below a certain amount compared to the value of your loan, you can get [liquidated](/borrowing-module/vaults/liquidations.md). 
+
+![Vaults lists](../../.gitbook/assets/vaults-list2.png)
+
+{% hint style="info" %}
+You have the possibility to use ETH and stETH instead of wETH and wstETH. In this case, transactions will require two permit or approval transactions: one to approve the wrapping of your tokens by the router contract, and another one for the actual transaction. 
+{% endhint %}
+
+## Add collateral and borrow agEUR
+
+To borrow agEUR, you need to **deposit** collateral tokens into a **vault**. Different vaults accept different tokens which have their specific loan-to-value (LTV). This means that you are able to borrow up to a certain amount of stablecoins from the amount deposited.  
+
+For example, wETH LTV is at 84% meaning that if you deposit 1,000 € worth of wETH, you can borrow up to 840 agEUR from this vault. 
+
+You can borrow agEUR in the same transaction that you deposit collateral to your vault. Once a vault is created, it is possible to deposit more collateral without borrowing more stablecoins. 
+
+Here are the steps to follow to deposit collateral and borrow agTokens: 
+1. Select the type of vault to create, defined by the collateral and stablecoin tokens
+2. Select the `Add/Borrow` action
+3. Enter the amount of collateral you want to deposit in the first input. 
+4. Enter the amount of stablecoins you want to borrow in the second input. 
+A summary of the changes on your vault and wallet will be displayed on the right. 
+5.  Add the actions to go to the next step. You will find a summary of your transaction. 
+6.  Click on `Approve` to approve to be used by the protocol through a permit signature or an approval transaction. 
+7. (optional) If using non-wrapped tokens, approve the router contract to interact with your vault to do the transaction. 
```

Comment:
If your transaction requires a wrapping, you'll need to sign a permit for the router contract to interact with your vault and perform the desired transaction